objects/app/apps.go

```diff
@@ -148,6 +148,21 @@ func (a *appsCmd) create(c *cli.Context) error {
 	}
 
 	appWithFlags(c, app)
+	// If architectures flag is not set then default it to nil
+	if c.IsSet("architectures") {
```

Comment:
(minor) Can we have 'architectures' in a const_variable? Helps to control the changes to parameter.
( I see that other parameters are used in the same way, but its a better coding practice. Your call anyways :) )

Reply:
Done